starting proxysql Error: <jemalloc>: Unsupported system page size (65536)

- OS version: CentOS Linux 7.7 (AltArch)
```
[root@centos7 ~]# uname -ms
Linux aarch64
[root@centos7 ~]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (AltArch)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (AltArch)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```
- ProxySQL version: 2.0.15
```
[root@centos7 ~]# rpm -qi proxysql
Name        : proxysql
Version     : 2.0.15
Release     : 1
Architecture: aarch64
Install Date: Mon 18 Jan 2021 02:49:08 PM CST
Group       : Development/Tools
Size        : 47916650
License     : GPL+
Signature   : RSA/SHA256, Fri 30 Oct 2020 12:16:32 AM CST, Key ID 04a562fb79953b49
Source RPM  : proxysql-2.0.15-1.src.rpm
Build Date  : Thu 29 Oct 2020 04:12:32 AM CST
Build Host  : e390c4f8baed
Relocations : (not relocatable)
URL         : https://proxysql.com/
Summary     : A high-performance MySQL proxy
Description :
A high-performance MySQL proxy
```
- The steps to reproduce the issue:
```
[root@centos7 ~]# /usr/bin/proxysql
<jemalloc>: Unsupported system page size
<jemalloc>: Error in malloc(): out of memory
Aborted (core dumped)

[root@centos7 ~]# getconf -a | grep -i pagesize
PAGESIZE                           65536
```

- jemalloc EXEC_PAGESIZE and OS Pagesize match:
```
[root@centos7 ~]# which proxysql
/usr/bin/proxysql
[root@centos7 ~]# strings /usr/bin/proxysql | grep -i exec_pagesize
EXEC_PAGESIZE 65536
[root@centos7 ~]# getconf -a | grep -i pagesize
PAGESIZE                           65536
[root@centos7 ~]# proxysql
2021-01-18 17:52:16 [INFO] Using config file /etc/proxysql.cnf
2021-01-18 17:52:16 main.cpp:1641:main(): [ERROR] Current RLIMIT_NOFILE is very low: 1024 .  Tune RLIMIT_NOFILE correctly before running ProxySQL
2021-01-18 17:52:16 main.cpp:1648:main(): [WARNING] Automatically setting RLIMIT_NOFILE to 4096
2021-01-18 17:52:16 [INFO] Using OpenSSL version: OpenSSL 1.1.1g  21 Apr 2020
2021-01-18 17:52:16 [INFO] SSL keys/certificates found in datadir (/var/lib/proxysql): loading them.
Daemon already running on PID file 57491
```

--- a/deps/Makefile
+++ b/deps/Makefile
@@ -1,19 +1,33 @@
 
 UNAME_S := $(shell uname -s)
-
-PROXYDEBUG := $(shell echo $(PROXYDEBUG))
-ifeq ($(PROXYDEBUG),1)
-MYCFLAGS=-O0
-MYJEOPT=--enable-xmalloc --enable-prof --enable-fill --enable-debug
-else
-MYCFLAGS=-O2
-MYJEOPT=--enable-xmalloc --enable-prof
-endif
+UNAME_MS := $(word 2, $(shell uname -ms))
 
 ifneq (,$(wildcard /etc/system-release))
 	CENTOSVER := $(shell rpm --eval %rhel)
 else
 	CENTOSVER := Unknown
+endif
+
+IS_ARM = $(if $(findstring aarch64, $(arch)),true,false)
+IS_CENTOS = $(if $(findstring Unknown, $(CENTOSVER)),false,true)
+
+PROXYDEBUG := $(shell echo $(PROXYDEBUG))
+ifeq ($(PROXYDEBUG),1)
+MYCFLAGS=-O0
+MYJEOPT=--enable-xmalloc --enable-prof --enable-fill --enable-debug
+
+ifeq ($(and $(IS_ARM),$(IS_CENTOS)),true)
+	MYJEOPT+=--with-lg-page=16
+endif
+
+else
+MYCFLAGS=-O2
+MYJEOPT=--enable-xmalloc --enable-prof
+
+ifeq ($(and $(IS_ARM),$(IS_CENTOS)),true)
+	MYJEOPT+=--with-lg-page=16
+endif
+
 endif
 
 PROXYSQLCLICKHOUSE := $(shell echo $(PROXYSQLCLICKHOUSE))

